### ISSUE TITLE
Importing from local storage of printscreened cards is missing

### ISSUE THREAD

**Issue by atirage21** [GeorgeYT9769/cardabase-app#8]
I saw picture in setting about importing cards, but in my instalation of apk is missing. Good solution of importing have apk supercards which import from printscreens. Your application dont ask for acces to local storage. 

Drzim palce ;), ze to nahradi umierajuce stocards.

**Comment by GeorgeYT9769** [comment]
Yeah, it's true, but it's on the list of feature I will add later. 

The screenshots are outdated a bit, because those import and export tiles were in pre-release and they didn't have any functions. I made screenshots and mockups, removed the tiles from the release and released the app with the old screenshots 😅.

Diki moc, je super vidiet niekoho zo sk, ono to vzniklo ako side projekt ktory budem postupne updatovat...

**Comment by GeorgeYT9769** [comment]
Hey, now you can use QR Codes to quickly share your cards! But later I will add feature to import/export the whole database...

**Comment by GeorgeYT9769** [comment]
Yo wassup! I also added a system that you can use to export/import all your cards at once! Simply press the button to export them into Internal Storage>Downloads folder and copy the data inside the exported txt file into the import menu! Sounds complicated? It's not! It's just me describing it 😂. The update is live on the GitHub and it will take a while for the F-Droid.
